pkgs/development/python-modules/mdformat-wikilink/default.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  mdformat,
+  poetry-core,
+  pytest-cov,
+  pytestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "mdformat-wikilink";
+  version = "0.2.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "tmr232";
+    repo = "mdformat-wikilink";
+    rev = "v${version}";
+    hash = "sha256-KOPh9iZfb3GCvslQeYBgqNaOyqtWi2llkaiWE7nmcJo=";
+  };
+
+  build-system = [
+    poetry-core
+  ];
+
+  dependencies = [
+    markdown-it-py
+    mdformat
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "mdformat_wikilink" ];
+
+  meta = with lib; {
+    description = "mdformat plugin for ensuring that wiki-style links are preserved during formatting";
```

Shouldn't it be
```suggestion
    description = "Mdformat plugin for ensuring that wiki-style links are preserved during formatting";
```
?

---

Happy to change. Not sure what style guide says on this. Notice there isn't really any consistency here: https://search.nixos.org/packages?query=mdformat